### ISSUE TITLE
F-Droid build failed

### ISSUE THREAD

**Issue by linsui** [Swordfish90/Lemuroid#480]
**Disclamer**
In order to keep this bug tracker manageable, you need to keep a few things in mind:
* Any issue not following this template will be closed as invalid
* Is the issue reproducible on RetroArch with the same core? If so, it's very likely a core issue and the upstream core repository is a better place to file it
* Is the issue already opened? Try out a search before filing one.

**Describe the bug**
```
> Task :lemuroid-app:processFreeDynamicReleaseResources FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':lemuroid-app:processFreeDynamicReleaseResources'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
   > Android resource linking failed
     /builds/fdroid/fdroiddata/build/com.swordfish.lemuroid/lemuroid-app/src/main/res/values-ar-rSA/strings.xml:100:5-83: AAPT: error: resource string/الغاء (aka com.swordfish.lemuroid:string/الغاء) not found.
         
     /builds/fdroid/fdroiddata/build/com.swordfish.lemuroid/lemuroid-app/src/main/res/values-nl-rNL/strings.xml:100:5-86: AAPT: error: resource string/annuleer (aka com.swordfish.lemuroid:string/annuleer) not found.
         
     /builds/fdroid/fdroiddata/build/com.swordfish.lemuroid/lemuroid-app/src/main/res/values-fr-rFR/strings.xml:98:5-85: AAPT: error: resource string/annuler (aka com.swordfish.lemuroid:string/annuler) not found.
         
     /builds/fdroid/fdroiddata/build/com.swordfish.lemuroid/lemuroid-app/src/main/res/values-pt-rPT/strings.xml:100:5-86: AAPT: error: resource string/cancelar (aka com.swordfish.lemuroid:string/cancelar) not found.
         
     /builds/fdroid/fdroiddata/build/com.swordfish.lemuroid/lemuroid-app/src/main/res/values-zh-rTW/strings.xml:100:5-80: AAPT: error: resource string/取消 (aka com.swordfish.lemuroid:string/取消) not found.
         
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
* Get more help at https://help.gradle.org/
BUILD FAILED in 1m 37s
53 actionable tasks: 41 executed, 12 from cache
```

Could you please take a look? Thanks!

**Comment by Swordfish90** [comment]
Thank you very much for reporting this. I just released a new tag 1.14.1 which contains the fix for this translations issue.

**Comment by linsui** [comment]
Thanks!
